packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -4508,6 +4510,121 @@ export class WalletService {
 
     this.storage.removeTxConfirmationSub(this.copayerId, opts.txid, cb);
   }
+
+  simplexGetQuote(req): Promise<any> {
+    return new Promise((resolve, reject) => {
+      if (!req.body.env || (req.body.env != 'sandbox' && req.body.env != 'production')) return reject(new Error('Simplex\'s request wrong environment'));
+
+      const API = config.simplex[req.body.env].api;
+      const API_KEY = config.simplex[req.body.env].apiKey;
+      const ip = req.headers['x-real-ip'] || req.connection.remoteAddress || req.ip;
```

Comment:
checking the code of `express-rate-limit.js` it seems `const ip = req.ip` is enough

Reply:
I was reading that there are different ways in which the ip can come. I did a utils function that covers all cases